plugins/templates/create.js

```diff
@@ -51,17 +58,28 @@ module.exports = () => ({
                     throw boom.unauthorized('Not allowed to publish this template');
                 }
 
-                // If template name exists and has good permission, check the exact version
-                return templateFactory.get({ name, version })
-                    .then((template) => {
-                        // If the exact version exists, throw 409
-                        if (template) {
-                            throw boom.conflict(`Template ${name}@${version} already exists`);
-                        }
+                // If template name exists and has good permission, get the latest version for major and minor
+                const searchVersion = minor ? `${major}${minor}` : major;
```

Comment:
with `1.2`, `getTemplate`returns `1.2.latestPatch`, 
with `1.2.0`, `getTemplate` returns the exact version `1.2.0`